typesafe_conductr_cli/shazar.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
```

Comment:
I know this was in the previous file, but we do not need this anymore. Executable file is being generated by pip.